File: RxDemo/app/src/main/java/com/mksk/rxdemo/data/source/interf/Repository.java
package com.mksk.rxdemo.data.source.interf;

import androidx.paging.PagingData;

import com.mksk.rxdemo.data.network.AppApi;
import com.mksk.rxdemo.data.network.response.Datum;

import io.reactivex.rxjava3.core.Flowable;

public abstract class Repository implements AppApi {
    public abstract Flowable<PagingData<Datum>> pagingDatum();
}

File: Client/app/src/main/java/com/mksk/client/data/source/interf/Repository.kt
package com.mksk.client.data.source.interf

import com.mksk.client.data.database.daos.revenue.RevenueDao
import com.mksk.client.data.database.daos.revenuedetail.RevenueDetailDao

interface Repository: RevenueDao, RevenueDetailDao {
}